polaris/hub/client.py

```diff
@@ -628,9 +683,12 @@ def upload_dataset(
                     if_exists=if_exists,
                 )
 
+        base_artifact_url = (
+            "datasets" if artifact_type == ArtifactType.STANDARD.value else "/competition/datasets"
```

Comment:
**Suggestion:** This is a comment for the Hub PR, but I'll leave it here for now so I don't forget. Wouldn't it be more consistent to use: 

```suggestion
            "datasets" if artifact_type == ArtifactType.STANDARD.value else "/competitions/datasets"
```

?

Reply:
Benchmark routes are just `/v1/benchmark/*`, so we followed the same here!